developer-tools.md

```diff
@@ -435,6 +435,44 @@ Error:(147, 9) value q is not a member of StringContext
         q"""
         ^ 
 ```
+<h4>Debug Spark with SBT and IntelliJ</h4>
+Debug Spark directly inside IntelliJ is somehow difficult. But with remote debug functionality which provided by IntelliJ,
```

Comment:
Let's remove this, "Debug Spark directly inside IntelliJ is somehow difficult." because adding a guide is enough.